assets/js/role_slider.js

```diff
@@ -0,0 +1,12 @@
+const roleSliderContainer = document.getElementById("roleSlider");
+const roleSliderTabs = document.querySelectorAll(".tab");
+roleSliderTabs[0].children[0].classList.add('selected');
+
+roleSliderContainer.addEventListener('mousedown', (e) => {
+    roleSliderTabs.forEach(tab => {
+        tab.children[0].classList.remove('selected');
+        if (tab.contains(e.target)) {
+            tab.children[0].classList.add('selected');
+        };
+    });
+});
```

Comment:
Out of curiosity, why the JS? From what I remember, the role slider needed no js for either theming or operation.

Reply:
Ah yes, so I noticed that in the mockups the "selected" tab has the same background image as the body of the role slider. I think this is to visually show to the user which tab they have selected when they click around them. I was trying to figure out if there was a way using hugo to add this "selected" class name.

My knowledge on hugo is very basic so I thought maybe it would be easier to just some JS to add that class instead.

Reply:
Oh, I see what I missed: if you'll look [here](https://github.com/The-Balance-FFXIV/glam/blob/9a716f4ae94b847536235643b977c2c6a60da74d/assets/css/main.css#L95), I commented out the styling for tabs which are "checked". With that, you should be able to remove all the js and the tabs become a css-only affair (which I'll fully admit I was needlessly proud of).

Try uncommenting out that section and moving your styles there?

Reply:
Will have an update in the next PR!

Reply:
This has been updated in the latest PR!